oli/src/commands/cli.rs

```diff
@@ -35,3 +68,33 @@ fn cli() -> App<'static> {
 
     app
 }
+
+enum CopyPath {
+    File(String),
+    Profile((String, String)),
+}
+
+fn build_object(
```

Comment:
I expect something like:

```
fn parse_path(path: &str) -> (Operator, String)
```

In which we will 

- parse `s3://abc` into a s3 operator (use data from `OliProfileGroup`) and path `/abc`.
- special case for local fs: if input doesn't have a scheme, like `path/to/dir`, we can build an `fs` operator directly.

In this way, we don't need to do hacks in cli main.

Reply:
Thanks for review!
I made some refactor, PTAL~